Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM docker.io/library/golang:1.20.2-alpine3.17 as builder
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022 Dell Inc, or its subsidiaries.
 
-WORKDIR /build
+FROM docker.io/library/golang:1.20.2-alpine as builder
```

Comment:
Any reason why the alpine version isn't fixed to match the second stage build?

Reply:
no reason. just matching the dockerfiles across all repos...
- https://github.com/opiproject/opi-spdk-bridge/blob/main/Dockerfile
- https://github.com/opiproject/opi-marvell-bridge/blob/main/Dockerfile
- https://github.com/opiproject/opi-nvidia-bridge/blob/main/Dockerfile
and others...